roles/etcd_common/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 # etcd server vars
-etcd_conf_dir: /etc/etcd
+etcd_conf_dir: "{{ '/etc/etcd' if not openshift.common.is_etcd_system_container else '/var/lib/etcd/etcd.etcd/etc'  }}"
+etcd_system_container_conf_dir: /var/lib/etcd/etc
```

Comment:
These variables seem to conflict a bit.

Any reason not to just use /etc/etcd for the system container? I worry about using alternative paths if we do not absolutely need to.